content/en/docs/pipelines/installation/localcluster-deployment.md

```diff
@@ -262,23 +266,42 @@ To set up access to your WSL instance:
 
 1. Copy `/etc/rancher/k3s/k3s.yaml` from WSL to `%HOME%.kube\config`.
 
-2. Edit the copied file by changing the server URL from `https://localhost:6443` 
-to the IP of the your WSL instance (`ip addr show dev eth0`) (For example, 
-`https://192.168.170.170:6443`.)
+2. Edit the copied file by changing the server URL from `https://localhost:6443`
+   to the IP of the your WSL instance (`ip addr show dev eth0`) (For example,
+   `https://192.168.170.170:6443`.)
+
+3. Run kubectl in a Windows terminal. If you don't kubectl installed, follow the
+   official [Kubernetes on Windows instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows).
+
+## K3ai [*alpha*]
+
+K3ai is a lightweight "infrastructure in a box" designed specifically to install
```

Comment:
I really like this initiative, thank you for your efforts!

Reply:
If I understood correctly, K3ai deploys KFP by default, does it?
If it deploys KFP by default, then we need to restructure the doc to make it clearer that k3ai isn't an equivalent of the other ways to install a local k8s. Users can skip installing KFP by themselves.

Reply:
Yes, [k3ai](https://kf5ai.gitbook.io/k3ai/) by default installs the kubeflow pipelines. But the basic idea is to handle different [plugins](https://github.com/kf5i/k3ai-plugins) and options like for example:

```
curl -sfL https://github.com/kf5i/k3ai/releases/latest/download/install | bash -s -- --gpu --plugin_argo
```

So basically the _default_ is:
```
curl -sfL https://github.com/kf5i/k3ai/releases/latest/download/install | bash -s --cpu --pipelines
```

We are working a plugin to install a full kubeflow deploy

Reply:
Just to clarify, at the moment we only install the pipelines, and not kfp.  We are thinking about a kfp plugin! .. working on it :)!

Reply:
> If it deploys KFP by default, then we need to restructure the doc to make it clearer that k3ai isn't an equivalent of the other ways to install a local k8s. Users can skip installing KFP by themselves.

Thanks for reviewing the doc update and all the suggestions @Bobgy. Maybe if we make it even more explicit in the first paragraph that K3ai is an "infrastructure in a box" that installs KFP and a few other things by default, it would be better? WDYT @alfsuse @Gsantomaggio 

For example:

```
K3ai is a lightweight "infrastructure in a box" designed specifically to install
- and configure AI tools and platforms on portable hardware, such as laptops and
- edge devices. This enables users to perform quick experimentations with Kubeflow
on a local cluster.
+ and configure Kubeflow Pipelines alongside other tools by running one command line on
+ portable hardware, such as laptops and edge devices. This enables users to
- perform quick experimentations with Kubeflow on a local cluster.
+ perform quick experimentations on a local cluster.
```

Or shall we say "a few command lines as possible" (less friction) instead of one command line "... or something along those lines?

(I know "install Kubeflow" is coming soon.)

Also, let's refactor this sentence:

```diff
- K3ai's main goal is to provide a quick way to install Kubernetes (K3s-based) and
+ The user benefits from having a quick way to install Kubernetes (K3s-based) and
 Kubeflow Pipelines with NVIDIA GPU support and TensorFlow Serving with just one
 line. (For Kubeflow and other component support, check [K3ai's
 website](https://kf5ai.gitbook.io/k3ai/#components-of-k-3-ai) for updates.) To
```

LMKWYT

Reply:
(My reference to KFP was because of this information: "a quick way to install Kubernetes (K3s-based) and Kubeflow Pipelines...")

Reply:
Fixed based on @Bobgy   suggestion.
Around k3ai we are working to find the simplest solution to avoid any installation and complexity to the user so I would suggest (since I need to write the steps based on @Gsantomaggio code) to merge this PR and I'll open another one where we explain how to use k3ai+kfp+pipelines  in a "click" :)

 ..btw it would be great to document how to use k3ai on Google as well it would be super cool!

Reply:
> the pipelines, and not kfp.

What's the difference?

Reply:
Oh I think what @Gsantomaggio meant is the sdk kfp. Currently we install pipelines stand-alone (based on Argo) but we are targeting also kubeflow complete and tekton as well.
Because to experiment you also would need the sdk kfp we are implementing a solution to get that installed automatically for the user (i.e. virtual env or a container with Jupiter and the sdk).

Reply:
Thanks for the explanation!